core/store/src/trie/mod.rs

```diff
@@ -1000,13 +1009,13 @@ mod tests {
         let root = test_populate_trie(&tries, &Trie::EMPTY_ROOT, ShardUId::single_shard(), changes);
         let trie = tries.get_trie_for_shard(ShardUId::single_shard(), root.clone());
         let mut iter = trie.iter().unwrap();
-        iter.seek(&vec![0, 116, 101, 115, 116, 44]).unwrap();
+        iter.seek_prefix(&vec![0, 116, 101, 115, 116, 44]).unwrap();
```

Comment:
While at it, let’s remove the unnecessary vector:

```suggestion
        iter.seek_prefix(&[0, 116, 101, 115, 116, 44]).unwrap();
```